rllib/policy/policy.py

```diff
@@ -158,7 +159,7 @@ def compute_single_action(self,
         if state is not None:
             state_batch = [[s] for s in state]
 
-        [action], state_out, info = self.compute_actions(
+        batched_action, state_out, info = self.compute_actions(
```

Comment:
This was the bug: Assuming that returned action was a batched "primitive" (but could be arbitrarily nested (dict/tuple) as well).